pkg/crypto/tinkcrypto/primitive/bbs/subtle/bls12381g2_verifier.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package subtle
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/doc/bbs/bbs12381g2pub"
```

Comment:
I assume there is a todo to move these primitives?

Reply:
not really, the subtle package uses these primitives directly

Reply:
unless we move the content of `"github.com/hyperledger/aries-framework-go/pkg/doc/bbs/bbs12381g2pub"` to `"github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/bbs/subtle/"` and make the core primitive internal (hidden within subtle package)

Reply:
In any case, I think it would make sense for crypto primitives to be in the crypto folder hierarchy rather than the doc folder hierarchy. Can be a followup.

Reply:
Sure, we can move them to crypto in a follow-up issue.